da-project-backend/app/api.py

```diff
@@ -55,14 +56,27 @@ def get_all_reports(
 
 @app.post("/api/report")
 def add_report(
-    report: ReportCreate,
+    report: Annotated[ReportCreate, File()],
     session: SessionDep,
-):
-    db_report = report.validate_to_report()
+) -> ReportWithColumnsResponse:
+    db_report, labels, rows, dtypes = ReportCreate(
+        name=report.name,
+        overview=report.overview,
```

Comment:
kaya ba na i-remove to sa request? sa response lang siya makita, autogenerated via gemini

Reply:
by "to" what do you mean?

Reply:
rn kasi required si overview sa isang post request, can we remove that? expected payload lang ay name and uploaded file

Reply:
done, thanks

Reply:
misunderstood your request. I thought you didn't want it there period but now I get that the overview needs to be generated by gemini. mbmb.

Reply:
done. please check the updated pr description and demos. both `report` and `page` creation does not require overview because it is generated by gemini.